src/modules/Dashboard/features/Applet/Schedule/Calendar/Toolbar/Toolbar.tsx

```diff
@@ -71,6 +71,7 @@ export const Toolbar = ({
       sx={{
         ...(isSelectedFutureDate() && { marginRight: theme.spacing(1) }),
         ...(isSelectedPastDate() && { margin: theme.spacing(0, 1) }),
+        height: '3.8rem',
```

Comment:
pls move this style to StyledTodayBtn
we have an agreement that if there are more than 3 styles item in sx, we use styled component

Reply:
Done. Thanks